### PR TITLE
Add styling and mock content to desktop view for actions page

### PR DESCRIPTION
**Pull request by tiffam**
Add an element on Actions view to toggle between two different actions pages temporarily as mockup is missing a link for user to go from Actions Preview to Actions.

screenshot 2018-10-27 10 47 37

Added styling and mock content (only for desktop view)
screenshot 2018-10-27 10 47 54

### CODE REVIEW

File: src/views/Actions.vue [hunk 1]
```diff
@@ -6,18 +6,57 @@
       <a href="#/results">Results</a>
     </div>
   </div>
-      <div class="results-message-wrapper">
-        <p>Earth is about the daily actions that enables responsible living.</p>
-        <p>Based on your responses, we've complied a list of actions you can do in Singapore...</p>
+    <div class="first-page-content" v-if="firstPage">
+      <p>Earth is about the daily actions that enables responsible living.</p>        
+      <p>Based on your responses, we've complied a list of actions you can do in Singapore...</p>
+      <div class="next" @click="changePage"><v-icon class="next-content" name="arrow-right"/></div>
```
Comment (SheldonCheng) [inline src/views/Actions.vue]: Instead of using `<div>` here, can consider using a `<button>`

File: src/views/Actions.vue [hunk 2]
```diff
@@ -6,18 +6,57 @@
       <a href="#/results">Results</a>
     </div>
   </div>
-      <div class="results-message-wrapper">
-        <p>Earth is about the daily actions that enables responsible living.</p>
-        <p>Based on your responses, we've complied a list of actions you can do in Singapore...</p>
+    <div class="first-page-content" v-if="firstPage">
```
Comment (yishus) [inline src/views/Actions.vue]: can consider calling this `actions-intro-message`

File: src/views/Actions.vue [hunk 3]
```diff
@@ -6,18 +6,57 @@
...
+      <p>Earth is about the daily actions that enables responsible living.</p>        
+      <p>Based on your responses, we've complied a list of actions you can do in Singapore...</p>
+      <div class="next" @click="changePage"><v-icon class="next-content" name="arrow-right"/></div>
+    </div>
+    <div class="second-page-content" v-else>
```
Comment (yishus) [inline src/views/Actions.vue]: can consider calling this `actions`/`actions-pledge`

later when this grows to more cards, move it into its own component

File: src/views/Actions.vue [hunk 4]
```diff
@@ -6,18 +6,57 @@
...
 <script type="text/javascript">
 
+
 export default {
     name: 'actions',
+    components: {
+   },
+    data: function() {
+    return {
+      firstPage: true,
```
Comment (yishus) [inline src/views/Actions.vue]: `showIntroMessage`

**Pull request merged**